conf/deployment/vsphere/upi_1az_rhel_vmfs_3m_3w.yaml

```diff
@@ -20,8 +20,8 @@ ENV_DATA:
   # This is used just for RHCOS nodes which will get removed after adding
   # RHEL nodes, so no need to have big number here.
   compute_memory: '16384'
-  # Once we will have DC which can handle 64GB we should use those and update here
   rhel_memory: '32768'
+  rhel_memory: '65536'
```

Comment:
There are duplicated key present `  rhel_memory: '32768'` should be removed

Reply:
DONE, Thanks fro noticing that